sdk/apimanagement/Microsoft.Azure.Management.ApiManagement/tests/ManagementApiTests/AuthorizationProviderTests.cs

```diff
@@ -0,0 +1,374 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+// using ApiManagement.Management.Tests;
+
+using Microsoft.Azure.Management.ApiManagement.Models;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using System;
+using System.IO;
+using System.Net;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ApiManagement.Tests.ManagementApiTests
+{
+    public class AuthorizationProviderTests : TestBase
+    {
+        [Fact]
+        [Trait("owner", "loganzipkes")]
+        public async Task CreateListUpdateDelete()
+        {
+            Environment.SetEnvironmentVariable("AZURE_TEST_MODE", "Playback");
```

Comment:
Did you change this to 
`Environment.SetEnvironmentVariable("AZURE_TEST_MODE", "Record");`

Reply:
@solankisamir I did